starter-code/bin/www

```diff
@@ -12,7 +12,7 @@ var http = require('http');
  * Get port from environment and store in Express.
  */
 
```

Comment:
Could you please leave it as 3000 in the future?  It makes it harder to grade like this.